### PR TITLE
Proposal of a alignment of BOT ontology and SAREF + SAREF4BLDG

### PR DESCRIPTION
**Pull request by GeorgFerdinandSchneider**
Please find the the alignment in the file.

### CODE REVIEW

**Comment by GeorgFerdinandSchneider** [comment]
I am afraid this has not yet been merged

File: bot/SAREFAlignment.ttl [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+
+#################################################################
+#    Classes
+#################################################################
+
+<https://w3id.org/bot#Building> owl:equivalentClass <https://w3id.org/def/saref4bldg#Building> .
+<https://w3id.org/bot#Space> owl:equivalentClass <https://w3id.org/def/saref4bldg#BuildingSpace> .
+<https://w3id.org/bot#Element> rdfs:subClassOf <https://w3id.org/def/saref4bldg#PhysicalObject> .
```
Comment (maximelefrancois86) [inline bot/SAREFAlignment.ttl:62]: I think bot:Element should be subClass of PhysicalObject, which is intended to be very general and correspond to dul:PhysicalObject

File: bot/SAREFAlignment.ttl [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+<https://w3id.org/bot#hasSpace> owl:equivalentProperty <https://w3id.org/def/saref4bldg#hasSpace> .
+
+
+
+#################################################################
+#    Classes
+#################################################################
+
+<https://w3id.org/bot#Building> owl:equivalentClass <https://w3id.org/def/saref4bldg#Building> .
+<https://w3id.org/bot#Space> owl:equivalentClass <https://w3id.org/def/saref4bldg#BuildingSpace> .
```
Comment (maximelefrancois86) [inline bot/SAREFAlignment.ttl:61]: ok

File: bot/SAREFAlignment.ttl [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+<https://w3id.org/bot#containsElement> owl:equivalentProperty <https://w3id.org/def/saref4bldg#contains> .
+<https://w3id.org/bot#hasSpace> owl:equivalentProperty <https://w3id.org/def/saref4bldg#hasSpace> .
+
+
+
+#################################################################
+#    Classes
+#################################################################
+
+<https://w3id.org/bot#Building> owl:equivalentClass <https://w3id.org/def/saref4bldg#Building> .
```
Comment (maximelefrancois86) [inline bot/SAREFAlignment.ttl:60]: ok

File: bot/SAREFAlignment.ttl [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+foaf:Person a owl:Class .
+foaf:name a owl:DatatypeProperty .
+
+											   
+#################################################################
+#    Object Properties
+#################################################################
+
+<https://w3id.org/bot#containsElement> owl:equivalentProperty <https://w3id.org/def/saref4bldg#contains> .
+<https://w3id.org/bot#hasSpace> owl:equivalentProperty <https://w3id.org/def/saref4bldg#hasSpace> .
```
Comment (maximelefrancois86) [inline bot/SAREFAlignment.ttl:52]: domain of bot:hasSpace is a Storey. This is more specific than domain of saref4bldg:hasSpace (which is a building or a building space).

if we consider that a storey is a building space, then bot:hasSpace is a sub property of saref4bldg:hasSpace

File: bot/SAREFAlignment.ttl [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+vs:term_status a owl:AnnotationProperty .
+foaf:Person a owl:Class .
+foaf:name a owl:DatatypeProperty .
+
+											   
+#################################################################
+#    Object Properties
+#################################################################
+
+<https://w3id.org/bot#containsElement> owl:equivalentProperty <https://w3id.org/def/saref4bldg#contains> .
```
Comment (maximelefrancois86) [inline bot/SAREFAlignment.ttl:51]: ok

**Comment by pipauwel** [comment]
The BOT ontology is maintained in a separate repo:
https://github.com/w3c-lbd-cg/bot
Alignments with SAREF are included there.

**Comment by GeorgFerdinandSchneider** [comment]
Correct: Has been merged already!

solved

**Pull request closed without merging**